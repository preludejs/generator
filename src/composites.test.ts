import * as G from './index.js'

test('42 composites', () => {
  expect(G.pipe1(
    G.composites(),
    G.take(42),
    G.array
  )).toMatchSnapshot()
})
