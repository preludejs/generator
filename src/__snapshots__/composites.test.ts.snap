// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`42 composites 1`] = `
Array [
  4,
  6,
  8,
  9,
  10,
  12,
  14,
  15,
  16,
  18,
  20,
  21,
  22,
  24,
  25,
  26,
  27,
  28,
  30,
  32,
  33,
  34,
  35,
  36,
  38,
  39,
  40,
  42,
  44,
  45,
  46,
  48,
  49,
  50,
  51,
  52,
  54,
  55,
  56,
  57,
  58,
  60,
]
`;
